tests/validate_json_format.py

```diff
@@ -169,20 +174,40 @@ def log_failure(self, code, line=None):
         self.fail = True
 
     def check_format(self, file_name):
-        """Initiate teh check."""
+        """Initiate the check."""
 
         self.fail = False
+        comment_align = None
         with codecs.open(file_name, encoding='utf-8') as f:
             count = 1
             for line in f:
+                indent_match = (RE_LINE_INDENT_TAB if self.use_tabs else RE_LINE_INDENT_SPACE).match(line)
+                end_comment = comment_align is not None and RE_COMMENT_END.search(line)
+                # Don't allow empty lines at file start.
                 if count == 1 and line.strip() == '':
                     self.log_failure(W_NL_START, count)
+                # Line must end in new line
                 if not line.endswith('\n'):
                     self.log_failure(W_NL_END, count)
+                # Trailing spaces
                 if RE_TRAILING_SPACES.match(line):
                     self.log_failure(W_TRAILING_SPACE, count)
-                if (RE_LINE_INDENT_TAB if self.use_tabs else RE_LINE_INDENT_SPACE).match(line) is None:
+                # Handle block comment content indentation
+                if comment_align is not None:
+                    if comment_align.match(line) is None:
+                        self.log_failure(W_COMMENT_INDENT, count)
+                    if end_comment:
+                        comment_align = None
+                # Handle general indentation
+                elif indent_match is None:
                     self.log_failure(W_INDENT, count)
+                # Enter into block comment
+                elif comment_align is None and indent_match.group(2):
+                    alignment = indent_match.group(1) if indent_match.group(1) is not None else ""
```

Comment:
How about something like `alignment = indent_match.group(1) or ""`

Reply:
I guess it's a stylistic thing for me.  I'm not checking its truthiness, I'm checking if it is None, so I am explicit about that. At least that holds true in this case. Part of it is habit as well, but here it is more illustrating what I am actually doing.